kalite/distributed/hbtemplates/exercise/software-keyboard.handlebars

```diff
@@ -0,0 +1,13 @@
+<button class='simple-button orange' id='show-keyboard'> {{_ "Hide Keypad" }}</button>
+
+<div class='container-fluid' id='software-keyboard'>
+    {{#each keys }}
+        <div class="row">
+            <div class="col-xs-12">
```

Comment:
Bootstrap buttons are designed to be displayed inline, so it makes sense to put them in the same div container.